src/YouTrackSharp/AgileBoards/Project.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Newtonsoft.Json;
+
+namespace YouTrackSharp.AgileBoards
```

Comment:
XML doc comments. Also is `id` the only property?

Reply:
Yes. To get more data on a project you need to use the https://www.jetbrains.com/help/youtrack/standalone/Get-Accessible-Projects.html API.